Source/Core/DolphinQt2/GameList/GameListModel.cpp

```diff
@@ -200,7 +200,7 @@ QSharedPointer<GameFile> GameListModel::GetGameFile(int index) const
   return m_games[index];
 }
 
-void GameListModel::UpdateGame(QSharedPointer<GameFile> game)
+void GameListModel::UpdateGame(const QSharedPointer<GameFile>& game)
```

Comment:
Nevermind, QList::push_back takes a const ref (and presumably makes an internal copy), so I think my original logic holds.